modules/aws_asg/prestart-hook.bash

```diff
@@ -12,6 +12,14 @@ main() {
     : "${reason:=not sure why}"
     echo "refusing to start travis-worker; instance imploding because ${reason}"
     echo "${reason}" | tee /var/tmp/travis-run.d/implode.confirm
+
+    curl \
+      -f \
+      -s \
+      -X POST
+      -H "Authorization: token $CYCLIST_AUTH_TOKEN" \
+      "$CYCLIST_URL/implosions/$(cat "${RUNDIR}/instance-id")"
+
```

Comment:
It's a minor difference, but I'd put this `curl` above the `tee` to `implode.confirm`.